env/netbox.env

```diff
@@ -15,6 +15,7 @@ EMAIL_USERNAME=netbox
 EMAIL_USE_SSL=false
 EMAIL_USE_TLS=false
 HOUSEKEEPING_INTERVAL=86400
+LANG=C.UTF-8
```

Comment:
Wouldn't it be better to add this as `ENV` instruction to the Dockerfile? Otherwise anyone not using our compose-based setup will have to define it.

Reply:
Yes, good idea.